server/plugin/command.go

```diff
@@ -615,6 +766,30 @@ func getAutocompleteData(config *Configuration) *model.AutocompleteData {
 
 	github.AddCommand(issue)
 
+	webhook := model.NewAutocompleteData("webhook", "[command]", "Available commands: add , list")
+	webhookList := model.NewAutocompleteData(list, "[owner/repo]", "List webhooks or an organization or repository.")
+	webhookList.AddTextArgument("Owner/repo to list webhooks from", "[owner/repo]", "")
+
+	webhook.AddCommand(webhookList)
+	webhookAdd := model.NewAutocompleteData(add, "[owner/repo] callback_url token content_type events insecure_ssl", "Add a webhook to desired owner[/repo] with optional --config")
+	webhookAdd.AddTextArgument("Owner/repo to create a webhook", "owner[/repo]", "")
+	webhookAdd.AddTextArgument("Call Back URL", "", "")
+	webhookAdd.AddTextArgument("token", "", "")
+	webhookAdd.AddTextArgument("content_type", "", "")
+	webhookAdd.AddTextArgument("Comma-delimited list of one or more of: create, delete, check_run, check_suite, code_scanning_alert, member, commit_comment, deploy_key, deployment_status, deployment, discussion_comment, discussion, fork, issue_comment, issues, label, meta, milestone, package, page_build, project_card, project_column, project, pull_request_review_comment, pull_request_review_thread, pull_request_review, pull_request, push, registry_package, release, repository, repository_import, repository_vulnerability_alert, secret_scanning_alert, star, status, team_add, public, watch, gollum . Defaults to *", "[features] (optional)", `/[^,-\s]+(,[^,-\s]+)*/`)
```

Comment:
Given that the token is optional, should these use `AddNamedTextArgument`?

Reply:
The arguments should also indicate what the default value is,

Reply:
@hanzei  token is not optional it is mandatory and the default value is empty , as it should be the web hook secret

Reply:
According to the GitHub UI the token is option. Does the API behave differently?

Reply:
yes it is optional in github , but the configuration we have , we need token as a mandatory field

Reply:
Can you please re-phrase your answer? I don't get about which configuration you are talking about.

Reply:
To access api with endpoint `plugins/github/webhook` we need to have the token orelse it will throw unauthorised

Reply:
Well, the command does allow configuring another callback URL right?